addons/sale/wizard/sale_make_invoice_advance.py

```diff
@@ -106,6 +97,27 @@ def _create_invoice(self, order, so_line, amount):
                 'analytic_account_id': order.analytic_account_id.id or False,
             })],
         }
+
+        return invoice_vals
+
+    def _get_payment_method_details(self, order):
```

Comment:
rename to `_get_advance_details`, payment method is ambiguous (other stuff named like this in odoo)

Reply:
This one is done :)